example/lib/mobile_scanner_overlay.dart

```diff
@@ -133,96 +132,199 @@
-// //   initState() {
-// //     super.initState();
-// //   }
-// //
-// //   @override
-// //   Widget build(BuildContext context) {
-// //     return Scaffold(
-// //       appBar: AppBar(
-// //         title: Text('Plugin example app'),
-// //       ),
-// //       body: Center(
-// //         child: Column(
-// //           crossAxisAlignment: CrossAxisAlignment.center,
-// //           mainAxisAlignment: MainAxisAlignment.center,
-// //           children: <Widget>[
-// //             Expanded(
-// //                 child: camState
-// //                     ? Center(
-// //                   child: SizedBox(
-// //                     width: 300.0,
-// //                     height: 600.0,
-// //                     child: MobileScanner(
-// //                       onError: (context, error) => Text(
-// //                         error.toString(),
-// //                         style: TextStyle(color: Colors.red),
-// //                       ),
-// //                       qrCodeCallback: (code) {
-// //                         setState(() {
-// //                           qr = code;
-// //                         });
-// //                       },
-// //                       child: Container(
-// //                         decoration: BoxDecoration(
-// //                           color: Colors.transparent,
-// //                           border: Border.all(
-// //                               color: Colors.orange,
-// //                               width: 10.0,
-// //                               style: BorderStyle.solid),
-// //                         ),
-// //                       ),
-// //                     ),
-// //                   ),
-// //                 )
-// //                     : Center(child: Text("Camera inactive"))),
-// //             Text("QRCODE: $qr"),
-// //           ],
-// //         ),
-// //       ),
-// //       floatingActionButton: FloatingActionButton(
-// //           child: Text(
-// //             "press me",
-// //             textAlign: TextAlign.center,
-// //           ),
-// //           onPressed: () {
-// //             setState(() {
-// //               camState = !camState;
-// //             });
-// //           }),
-// //     );
-// //   }
-// // }
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:mobile_scanner/mobile_scanner.dart';
+import 'package:mobile_scanner_example/barcode_scanner_window.dart';
+import 'package:mobile_scanner_example/scanner_error_widget.dart';
+
+void main() {
+  debugPaintSizeEnabled = false;
+  runApp(HomePage());
+}
+
+class HomePage extends StatefulWidget {
+  @override
+  HomeState createState() => HomeState();
+}
+
+class HomeState extends State<HomePage> {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(home: BarcodeScannerWithOverlay());
+  }
+}
+
+class BarcodeScannerWithOverlay extends StatefulWidget {
+  @override
+  _MyAppState createState() => _MyAppState();
```

Comment:
Could you rename the state class to include the name of the widget? This is a common pattern when creating StatefulWidgets.

```suggestion
class BarcodeScannerWithOverlay extends StatefulWidget {
  @override
   _BarcodeScannerWithOverlayState createState() => _BarcodeScannerWithOverlayState();
}

class _BarcodeScannerWithOverlayState extends State<BarcodeScannerWithOverlay> {
  // ...
}
```